ot/da.py

```diff
@@ -1114,7 +1255,10 @@ class SinkhornTransport(BaseTransport):
 
     def __init__(self, reg_e=1., mode="unsupervised", max_iter=1000,
                  tol=10e-9, verbose=False, log=False, mapping="barycentric",
-                 metric="sqeuclidean", distribution="uniform"):
+                 metric="sqeuclidean",
+                 distribution_estimation=distribution_estimation_uniform,
```

Comment:
Ok good. It is much simpler and allows the user to move around freely with the distribution estimation.